pkg/v1/providers/tests/unit/fixtures/tkg-bom-v1.4.0.yaml

```diff
@@ -318,6 +318,12 @@ components:
       tanzuStandardPackageRepositoryImage:
         imagePath: packages/standard/repo
         tag: v1.4.0
+  tanzu-framework-management-packages:
```

Comment:
Question: What purpose does this file serve? I'm not familiar with it. From the file name, it seems adding component to v1.4?

Reply:
This is just a fixture/testdata used for unit testing. We will have `tanzu-framework-management-packages` as a BOM entry, so this change prevents unit test from panicking.

Reply:
there are ytt unit tests and after adding CRS for the management package repo, they were failing, so added this component to that config file to fix the failing test..